test/test_sparse.py

```diff
@@ -4049,6 +4049,58 @@ def test_basic(self):
         self.assertEqual(r.values(), torch.empty(0, 4, device='meta'))
 
 
+class TestSparseAny(TestCase):
+
+    def test_generate_simple_inputs(self):
+        # Temporarily disable BSC and BSC layouts as these don't support select yet
+        layouts = [torch.strided, torch.sparse_coo, torch.sparse_csr, torch.sparse_csc, torch.sparse_bsr, torch.sparse_bsc][:-2]
+
+        generators = list(map(self.generate_simple_inputs, layouts))
+
+        tested_combinations = set()
+        while True:
+            try:
+                tensors = [next(g) for g in generators]
```

Comment:
Can't you use [`zip`](https://docs.python.org/3.3/library/functions.html#zip) for this? Or is that not the intend?

Reply:
Yes, using `zip` is a good idea.

Reply:
Done. Thanks for the suggestion, @cpuhrsch !